dispatch/apps/api/views.py

```diff
@@ -403,6 +404,10 @@ def get_queryset(self):
                 Q(description__icontains=q) |
                 Q(host__icontains=q)
             )
+        if pending:
+            queryset = queryset.filter(
+                Q(is_submission__exact=1)
```

Comment:
You don't need to use `Q` here since there's only one condition. Also, the default queryset should exclude submitted events:

```py
if pending:
    queryset = queryset.filter(is_submission=True)
else:
    queryset = queryset.filter(is_submission=False)
```

Can you add a test case for this as well?